public/hwcdefs.h

```diff
@@ -76,5 +76,10 @@ enum DisplayPowerMode {
                     // updates from the client
 };
 
+enum HWCColorTransform {
+  kIdentical = 0,
```

Comment:
Add documentation what do these different values mean, do these cover all cases we support ?

Reply:
The SurfaceFlinger only uses HAL_COLOR_TRANSFORM_IDENTITY and HAL_COLOR_TRANSFORM_ARBITRARY_MATRIX for color transform,  I think we can only support these 2 hints.

Reply:
ya lets just do that than